ci_scripts/github_release.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+VAR=0
```

Comment:
Does that pass `shellcheck`? I think it should be a lowercase variable tbh. Same for `FINAL`.